force-app/test/LDConfigTest.cls

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Is this comment needed?

Reply:
nope, removing. It is caused by auto generating the test class